### PR TITLE
Add a function to bind default arguments to pre-defined Selenium WebDriver functions

### PR DESCRIPTION
**Pull request by VovaViliLox**
Add a function to bind default arguments to pre-defined Selenium WebDriver functions, as per Melvil's request about re-usability of functions. This enables to [avoid code repetition in a scalable way](https://en.wikipedia.org/wiki/Don't_repeat_yourself).

### CODE REVIEW

File: lox_services/scraping/selenium_util.py [hunk 1]
```diff
@@ -99,6 +99,31 @@ def safe_find_element(
     return wait.until(element_present)
 
 
+def find(
+    driver: webdriver.Chrome, selector: Union[str, Enum], selector_type: By
```
Comment (natasazekic-loxsolution) [inline lox_services/scraping/selenium_util.py:103]: By.CSS_SELECTOR

File: lox_services/scraping/selenium_util.py [hunk 2]
```diff
@@ -99,6 +99,31 @@ def safe_find_element(
...
+    """
+    Same as 'safe_find_element', but without waiting. find(Some.CONSTANT) should be
+    less typing than driver.find_element(Some.CONSTANT.value), which results in cleaner code.
+
+    ## Arguments:
+    - `driver`: The driver that will be used to look up the element.
+    - `selector_type`: The type of the selector used (CSS by default). It can be any element of
+    the selenium.webdriver.common.by.By .
+    - `selector`: The selector used to look for the element.
+    - `timeout`: The maximum number of seconds to wait until the function returns a timeout.
```
Comment (natasazekic-loxsolution) [inline lox_services/scraping/selenium_util.py:114]: to timeout in arguments

File: lox_services/scraping/selenium_util.py [hunk 3]
```diff
@@ -169,7 +199,7 @@ def wait_then_clear(
         - `wait`: The wait element that will be used to wait until a certain number of second.
         - `selector`: The selector used to look for the elements.
     """
-    wait.until(lambda driver: driver.find_element(By.CSS_SELECTOR, selector))
+    wait.until(lambda dr: dr.find_element(By.CSS_SELECTOR, selector))
```
Comment (natasazekic-loxsolution) [inline lox_services/scraping/selenium_util.py:202]: please reverse this change

File: lox_services/scraping/selenium_util.py [hunk 4]
```diff
@@ -27,8 +28,7 @@ def wait_until_page_loaded(driver: webdriver.Chrome, timeout: int = 10):
     """
     try:
         WebDriverWait(driver, timeout).until(
-            lambda driver: driver.execute_script("return document.readyState")
-            == "complete"
+            lambda dr: dr.execute_script("return document.readyState") == "complete"
```
Comment (natasazekic-loxsolution) [inline lox_services/scraping/selenium_util.py:31]: please reverse this change

File: lox_services/scraping/selenium_util.py [hunk 5]
```diff
@@ -188,7 +218,7 @@ def wait_then_send_keys(
     - `input_text`: The text that will be sent to the element
     - `clear`: True if the field needs to be cleared before to get filled
     """
-    wait.until(lambda driver: driver.find_element(By.CSS_SELECTOR, selector))
+    wait.until(lambda dr: dr.find_element(By.CSS_SELECTOR, selector))
```
Comment (natasazekic-loxsolution) [inline lox_services/scraping/selenium_util.py:221]: please reverse this change

File: lox_services/scraping/selenium_util.py [hunk 6]
```diff
@@ -214,10 +244,11 @@ def safe_send_keys(
 def wait_till_disapear(
     wait: WebDriverWait, selector_type: By, selector: str, timeout: int = 60
```
Comment (natasazekic-loxsolution) [inline lox_services/scraping/selenium_util.py:245]: By.CSS_SELECTOR

**Review by natasazekic-loxsolution — APPROVED** [review]

File: lox_services/scraping/selenium_util.py (hunk 3 above)
Comment (VovaViliLox) [reply lox_services/scraping/selenium_util.py:202]: Will do, but it does shadow a name - https://stackoverflow.com/questions/20125172/what-is-the-problem-with-shadowing-names-defined-in-outer-scopes

File: lox_services/scraping/selenium_util.py (hunk 4 above)
Comment (VovaViliLox) [reply lox_services/scraping/selenium_util.py:31]: Will do, but it does shadow a name - https://stackoverflow.com/questions/20125172/what-is-the-problem-with-shadowing-names-defined-in-outer-scopes

File: lox_services/scraping/selenium_util.py (hunk 5 above)
Comment (VovaViliLox) [reply lox_services/scraping/selenium_util.py:221]: Will do, but it does shadow a name - https://stackoverflow.com/questions/20125172/what-is-the-problem-with-shadowing-names-defined-in-outer-scopes

**Pull request merged**